carma_ros2_utils/include/carma_ros2_utils/containers/containers.hpp

```diff
@@ -33,10 +33,13 @@ namespace containers
  * 
  * \param input The input vector to downsample
  * \param n The count of the elements to save
+ * \param include_last_point If true, append the last point
+          NOTE: default is true as this function is only used for trajectory generation 
+          and this keeps the lanelet's boundaries consistent no matter the downsample ratio
```

Comment:
While we're making an edit here, can we also add to the function description, or to the return description, that the downsampled vector will always include the first point of the input vector (at least when n > 0)?

Reply:
added